interfaces/Portalicious/src/app/utils/local-time-to-utc-time.ts

```diff
@@ -0,0 +1,10 @@
+export const localTimeToUtcTime = (localDate: Date): Date =>
+  new Date(
+    localDate.getUTCFullYear(),
+    localDate.getUTCMonth(),
+    localDate.getUTCDate(),
+    localDate.getUTCHours(),
+    localDate.getUTCMinutes(),
+    localDate.getUTCSeconds(),
+    localDate.getUTCMilliseconds(),
+  );
```

Comment:
Ah. not that I've checked it separately, this is better then the "shorter" version:

`new Date(localDate.toUTCString()).toISOString();`

As that 👆 doesn't take the miliseconds into account.